frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -644,14 +644,11 @@ pub mod pallet {
 		#[pallet::constant]
 		type SignedDepositWeight: Get<BalanceOf<Self>>;
 
-		/// The maximum number of voters to put in the snapshot. At the moment, snapshots are only
-		/// over a single block, but once multi-block elections are introduced they will take place
-		/// over multiple blocks.
-		///
-		/// Also, note the data type: If the voters are represented by a `u32` in `type
-		/// CompactSolution`, the same `u32` is used here to ensure bounds are respected.
+		/// The maximum number of electing voters to put in the snapshot. At the moment, snapshots
+		/// are only over a single block, but once multi-block elections are introduced they will
+		/// take place over multiple blocks.
 		#[pallet::constant]
-		type VoterSnapshotPerBlock: Get<SolutionVoterIndexOf<Self>>;
+		type ElectingVotersPerBlock: Get<SolutionVoterIndexOf<Self>>;
```

Comment:
I'm far away from being good at naming. However, I think this name can still be improved.

`MaxVotersPerBlock` why not take this? The comment seems to suggest that this is the maximum number of voters? And "electing voters" isn't that trying to convey the same twice? I mean a voter is electing or are there also non electing voters? If yes, why are they called voters?

Reply:
non-electing voters exist, and are those that do not make it into the snapshot. They are technically nominators that are not included in the election, thus are not influencing the election. 

Just to give a bit more context, I've written about it in length here: https://docs.google.com/document/d/1bisngPfte-P2vMuP_JrV6fIpNnd04uCWrqt4UbFklF8/edit#heading=h.kdyyexvr13fu

Reply:
Okay, then something like `MaxElectingVotersPerBlock`?

Reply:
Should work. 

I kinda wish it was a good argument to omit the `PerBlock` as well, since "per what else could it be if not per block"?, but I guess being explicit is better.

Reply:
Reading the comment again, I think dropping `PerBlock` should be fine!

Reply:
I still find this naming not perfect, but I approve it for you :P